static/js/stream_edit.js

```diff
@@ -489,6 +489,7 @@ export function initialize() {
         }
 
         stream_settings_ui.sub_or_unsub(sub);
+        window.location.reload();
```

Comment:
Instead of reloading the app, can't we just call `narrow.activate` with operands as per the view?

Reply:
Hi @sahil839 ! Thanks for the suggestions! I fixed the old issue and also the new one you mentioned here. In addition, I also noticed that 
` $("#compose-send-status").on("click", ".sub_unsub_button", (event) => {`
triggers the `sub_unsub_button` click event but couldn't find where exactly this button is... I tried with this button, but it turns out this one is already fixed by the commit.
<img width="717" alt="Screen Shot 2022-12-06 at 9 59 30 PM" src="https://user-images.githubusercontent.com/65670964/206077344-473b814b-9a89-4e84-8fb3-2219612b8eae.png">
Thanks!